helm/charts/infra/templates/connector/prometheusrule.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if include "connector.enabled" . | eq "true" }}
+{{- if and .Values.connector.metrics.enabled .Values.connector.metrics.prometheusRule.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "connector.fullname" . }}
+  labels:
+{{- include "connector.labels" . | nindent 4 }}
+{{- if .Values.connector.metrics.prometheusRule.labels }}
+{{- toYaml .Values.connector.metrics.prometheusRule.labels | nindent 4 }}
+{{- end }}
+  annotations:
+{{- toYaml .Values.connector.metrics.prometheusRule.annotations | nindent 4 }}
+spec:
+{{- $commonLabels := printf "job=%q, namespace=%q" (include "connector.name" .) .Release.Namespace }}
+  groups:
+    - name: connector.rules
+      rules:
+{{- with .Values.connector.metrics.prometheusRule.additionalRules }}
+{{- toYaml . | nindent 8 }}
+{{- end }}
+        - alert: InfraConnectorDown
+          annotations:
+{{- with .Values.connector.metrics.prometheusRule.additionalRuleAnnotations }}
+{{- toYaml . | nindent 12 }}
+{{- end }}
+            description: '{{ `{{` }} $labels.job {{ `}}` }} has disappeared from Prometheus.'
+            summary: Target disappeared from Prometheus.
+            runbook_url: https://infrahq.com/docs/runbooks/infraconnectordown
```

Comment:
This is a 404?

Reply:
Yes the runbook doesn't exist yet. Will leave it out before merging